_config.yml

```diff
@@ -53,9 +53,9 @@ phases:
       name:
         singular: fermée
         plural: fermées
-      description: Le service a été rendu obsolète par des évolutions techniques, règlementaires ou d’usage. Il a été fermé en lien avec les partenaires, les usagers et son éventuel successeur.
+      description: Les évolutions de la technologie, de la réglementation et des usages rendent le service obsolète. Il est fermé en lien avec les partenaires, les usagers et l'administration qui était susceptible  de le reprendre . 
     death:
       name:
         singular: abandonnée
         plural: abandonnées
-      description: Le service a été abandonné en cours de construction. L'abandon est expliqué pour capitaliser sur l'expérience accumulée.
+      description: Le service est abandonné en cours de construction. L'abandon est expliqué pour capitaliser sur l'expérience accumulée (PAS CONVAINCU PAR CETTE IDEE).
```

Comment:
Ok !
Je suis off aujourdui. Je revois ça lundi, mais on est quasiment bon non ? 
:)

De mon téléphone de marque quelconque

> Le 25 mars 2016 à 09:19, Matti Schneider <notifications@github.com> a écrit :
> 
> In _config.yml:
> 
> >      death:
> >        name:
> >          singular: abandonnée
> >          plural: abandonnées
> > -      description: Le service a été abandonné en cours de construction. L'abandon est expliqué pour capitaliser sur l'expérience accumulée.
> > +      description: Le service est abandonné en cours de construction. L'abandon est expliqué pour capitaliser sur l'expérience accumulée (PAS CONVAINCU PAR CETTE IDEE).
> C’est bien ça. Je ne sais pas à quel point il est évident qu’un postmortem est écrit et que la capitalisation est systématique. Ce n’est pas une pratique évidente dans tous les environnements  
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>